rocAL/rocAL/include/api/rocal_api_types.h

```diff
@@ -121,7 +118,8 @@ enum RocalTensorLayout
     ROCAL_NHWC = 0,
     ROCAL_NCHW = 1,
     ROCAL_NFHWC = 2,
-    ROCAL_NFCHW = 3
+    ROCAL_NFCHW = 3,
+    ROCAL_NONE = 4
```

Comment:
@fiona-gladwin what is the use case for this?

Reply:
This layout will be used for tensors that do not have a specific layout (other than image / video)

Reply:
Please remove ROCAL_NONE if it is not used in the code. This is confusing from the layout perspective